diagnostic.md

```diff
@@ -42,8 +57,12 @@ let weather;
 weather = "sunny";
 weather === "sunny";
 ```
+What are the values of these expressions?
+"sunny" which is a string. It is words in a quotation.
+
```

Comment:
The 3rd line is a Boolean expression that checks whether the variable `weather` is exactly equal to the string `"sunny"`. In this case, the value of this expression would be `true`.